tools/exporter/exporter.py

```diff
@@ -149,7 +168,7 @@ def load_setting_export_profile_mapping(logger, config_settings):
     """
     try:
         profile_mapping = {}
-        export_profile_settings = config_settings['export_profiles']
+        export_profile_settings = config_settings['export_options']['export_profiles']
```

Comment:
Are you changing the config file structure? If yes and it's a breaking change then 3.0 instead of 2.1 version